modules/swagger-codegen/src/main/resources/python/api.mustache

```diff
@@ -47,7 +24,7 @@ class {{classname}}(object):
             self.api_client = config.api_client
 {{#operation}}
 
-    def {{nickname}}(self, {{#sortParamsByRequiredFlag}}{{#allParams}}{{#required}}{{paramName}}, {{/required}}{{/allParams}}{{/sortParamsByRequiredFlag}}**kwargs):
+    def {{nickname}}(self{{#sortParamsByRequiredFlag}}{{#allParams}}, {{paramName}}{{#required}}{{/required}}{{^required}}=None{{/required}}{{/allParams}}{{/sortParamsByRequiredFlag}}, callback=None):
```

Comment:
Using your approach will make the method signature really long if there're many optional method arguments. Shall we keep the current approach using **kargs??